config-staging.json

```
@@ -89,6 +89,46 @@
       "supportedBy": [],
       "type": "EVM"
     },
+    {
+      "coinApi": "evm",
+      "explorerUrl": "https://arbiscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Arbitrum",
+      "nativeAsset": "ARB",
+      "networkTicker": "ARB",
```

```suggestion
      "networkTicker": "ARBETH",
```
I think this needs to match the coin-definitions entry: https://github.com/blockchain/coin-definitions/blob/master/scripts/statics.py#L69